infra/modules/dms/networking.tf

```diff
@@ -0,0 +1,10 @@
+# Add Security Groups for VPC -> DMS here
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+# This connection was created in the console, the provided arguments act more as a query than a reference
+data "aws_vpc_peering_connection" "dms" {
+  owner_id        = data.aws_caller_identity.current.account_id
+  cidr_block      = "172.31.0.0/16" # our cidr block
+  peer_cidr_block = "10.220.0.0/16" # their cidr blcok
+}
```

Comment:
_(non-blocking questions)_

1. has the peering connection been accepted on their end?
2. do we have direct access to their AWS account? eg. can we accept the peering request ourselves, or do we have to rely on someone at MH to perform that action?

Reply:
1. yes
2. From what I've seen, we don't have access. There's a coordination channel in Slack that we use for requests like this.